libs/sdk-ui-filters/src/AttributeFilter@next/AttributeFilterButton.tsx

```diff
@@ -16,7 +17,8 @@ export const AttributeFilterButton: React.FC<IAttributeFilterButtonProps> = (pro
         <AttributeFilterBase
             {...props}
             DropdownButtonComponent={props.DropdownButtonComponent ?? AttributeFilterDropdownButton}
-            LoadingComponent={props.LoadingComponent ?? LoadingComponent}
+            LoadingComponent={props.FilterLoading ?? props.LoadingComponent ?? LoadingComponent}
+            ErrorComponent={props.FilterError}
```

Comment:
Maybe we should check props for FilterError and if it is not defined than set default one ?